jerry-core/parser/js/opcodes-dumper.h

```diff
@@ -16,25 +16,238 @@
 #ifndef OPCODES_DUMPER_H
 #define OPCODES_DUMPER_H
 
-#include "opcodes.h"
 #include "ecma-globals.h"
 #include "lexer.h"
+#include "opcodes.h"
+#include "serializer.h"
 
-typedef enum __attr_packed___
+/**
+ * Operand (descriptor of value or reference in context of parser)
+ */
+class jsp_operand_t
```

Comment:
@akiss77, I'll provide measurements for both versions (i.e. before switching to class, and just after) today.

I think, we should perform the measurements on engine code, rather on some external tests / benchmarks. Because of this, I prepared the patch with usage of classes.

So, we could see if usage of classes could be ok.